pkg/signals/signal_posix.go

```diff
@@ -1,6 +1,6 @@
 // +build !windows
 
-/*
+/*Package signals provides basic signal's handling
```

Comment:
Can we remove the added comment here...
And just add a blank line between the comments and package declaration. 
I mean add a new line between line 17 and 18.
Thats all!